src/Symfony/Component/Security/Http/Firewall/BasicAuthenticationListener.php

```diff
@@ -67,7 +67,7 @@ public function handle(GetResponseEvent $event)
         }
 
         if (null !== $this->logger) {
-            $this->logger->info(sprintf('Basic Authentication Authorization header found for user "%s"', $username));
+            $this->logger->info(sprintf('Basic authentication Authorization header found for user "%s"', $username));
```

Comment:
What about moving the username as well (same below.) The more static the message is, the better for log aggregators.